integreat_cms/cms/templates/events/event_form_sidebar/minor_edit_box.html

```diff
@@ -7,14 +7,26 @@
     feather
 {% endblock collapsible_box_icon %}
 {% block collapsible_box_title %}
-    {% translate "Minor edit" %}
+    {% translate "Minor edit and automatic translations" %}
```

Comment:
According to the design (s. screenshot) the title is supposed to be different. In a chat with @osmers last week it was reinforced, that the new title is supposed to be the one from the design. 

![grafik](https://user-images.githubusercontent.com/72705147/218747970-8bd03ec6-95c8-4688-bc9b-5549d0354d04.png)
```suggestion
    {% translate "Translations" %}
```

Reply:
Thanks, I was not aware. Where did that discussion happen? Because I couldn't find it via the issue(s)

Reply:
Yes, that's unfortunate. The design (and therefore also the discussion) were made in Figma, and not everyone has access to the board. That is definitively not great

Reply:
Here is [the link](https://www.figma.com/file/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=166%3A1980) for Figma, in case you don't already have it

Reply:
Thanks for the link, I did not have it yet, I think.
Just so I'm clear (maybe @osmers can help out here), only already existing translations should be updated? New ones should not be created, even if the language is available through DeepL?

Reply:
@charludo correct - we will use the management section for creating new translations

Reply:
@charludo just discussed the title of the box as well - could you change it from "Übersetzungen" to "Auswirkungen auf andere Übersetzungen" and delete the subtitle (the one that is currently "Auswirkungen auf andere Übersetzungen")?

Reply:
> the one that is currently "Auswirkungen auf andere Übersetzungen"

AFAIK this changes dependent on if the language that is currently being edited is the root of the language tree ("Auswirkungen auf andere Übersetzungen"), a leaf ("Auswirkungen auf den Übersetzungsstatus"), or has both parent and child elements ("Auswirkungen auf den Status dieser und anderer Übersetzungen").

Do you want me to move this behaviour into the title of the box, or actually remove this feature and keep it to a static "Auswirkungen auf andere Übersetzungen"?

Reply:
> correct - we will use the management section for creating new translations

In that case, shouldn't there (eventually, not necessarily in this PR) be a second checkbox that translates to all child languages, at least for new pages/events/pois? I would imagine it's a lot more convenient to just check a box and hit "Save" as opposed to having to split this into two steps on different forms

Reply:
> > the one that is currently "Auswirkungen auf andere Übersetzungen"
> 
> AFAIK this changes dependent on if the language that is currently being edited is the root of the language tree ("Auswirkungen auf andere Übersetzungen"), a leaf ("Auswirkungen auf den Übersetzungsstatus"), or has both parent and child elements ("Auswirkungen auf den Status dieser und anderer Übersetzungen").
> 
> Do you want me to move this behaviour into the title of the box, or actually remove this feature and keep it to a static "Auswirkungen auf andere Übersetzungen"?

Could we maybe have a short call for this bcs I'm not sure I completely understand :D

Reply:
> > correct - we will use the management section for creating new translations
> 
> In that case, shouldn't there (eventually, not necessarily in this PR) be a second checkbox that translates to all child languages, at least for new pages/events/pois? I would imagine it's a lot more convenient to just check a box and hit "Save" as opposed to having to split this into two steps on different forms

We have actually discussed this a lot during the conference and changed a lot regarding what was planned. So we have three cases in which we need a translation and at least one of them requires a bulk action which will now be added in the page tree - but your suggestions is something we might think of later in case Kommunen complain about the bulk action for single pages. The only "problem" would be that we'd need a possibility to choose languages for new translations, which might be difficult/work with a pop up, if we incorporate it directly into the page form.

Reply:
@osmers thanks for letting me know about the changed plans, yes, I think a call would be great. I'll message you on Mattermost.